base/common/python/pki/nssdb.py

```diff
@@ -698,44 +702,36 @@ def __add_cert(
         check = True
 
         cmd = [
-            'pki',
-            '-d', self.directory
+            'certutil',
```

Comment:
Hmm.. the `__add_cert()` is a supposed to be a replacement candidate for `add_cert()` which will import a cert using `pki nss-cert-import` instead of `certutil` (to avoid its limitations). The plan is if `pki nss-cert-import` works as good as (or better than) `certutil`, the code in `__add_cert()` will replace the code in `add_cert()`.

Right now the `pki nss-cert-import` can import a cert without a nickname using JSS, but it still depends on `certutil` for importing a cert with a nickname: 
https://github.com/dogtagpki/pki/blob/master/base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java#L100
Ideally, if there's time, we want to add the missing functionality in JSS so `pki nss-cert-import` can import a cert using JSS in all cases.

Based on your tests it looks like `pki nss-cert-import` is not ready yet, but instead of modifying `__add_cert()` to use `certutil` (which defeats its purpose), is it possible to call `add_cert(use_jss=False)` instead?

Reply:
I can try it again but yeah, I should have at least consolidated on one certutil call. Not sure why I did that. I played with use_jss early a while ago.

I have the feeling that the JSS code does something additional under the hood because while certutil adds the cert ok to the database, when the OCSP cert is going to be issued the installer blows up because the CA cert can't be found by nickname. I'll keep poking.

Reply:
It was permissions. It's always permissions. I should have a new patch soon.